src/components/Registration/RegAddEditOrgPerson.vue

```diff
@@ -224,7 +224,7 @@
                 ref="mailingAddressNew"
                 class="mt-4 mb-n6"
                 :editing="true"
-                :schema="enableRules ? PersonAddressSchema : null"
+                :schema="PersonAddressSchema"
```

Comment:
Same fix as 11854 (which was on a side branch).

According to 11837, we'll review this later.